src/components/ShareStructure/share-mixin.ts

```diff
@@ -1,25 +1,15 @@
 import { Component, Vue } from 'vue-property-decorator'
 
 /**
- * Mixin that provides some useful common utilities.
+ * Mixin that provides some utilities.
  */
 @Component({})
-export default class CommonMixin extends Vue {
+export default class ShareMixin extends Vue {
```

Comment:
I like it.

Reply:
Can you update the comment (line 4) to say what this file is -- why it looks like Common Mixin but isn't?